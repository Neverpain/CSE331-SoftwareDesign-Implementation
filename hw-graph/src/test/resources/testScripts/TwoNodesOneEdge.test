# Tests that an edge can be added to a graph, and that
# a pair of nodes and a single edge connecting them
# can be printed out

# Create a graph
CreateGraph graph1

# Add two nodes
AddNode graph1 n1
AddNode graph1 n2

# Add one edge
AddEdge graph1 n1 n2 n1n2

# Print the nodes contained in the graph and the
# edges outgoing from the parentNodes
ListNodes graph1
ListChildren graph1 n1