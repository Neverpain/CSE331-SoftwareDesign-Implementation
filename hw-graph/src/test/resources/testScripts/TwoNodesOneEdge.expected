# Tests that an edge can be added to a graph, and that
# a pair of nodes and a single edge connecting them
# can be printed out

# Create a graph
created graph graph1

# Add two nodes
added node n1 to graph1
added node n2 to graph1

# Add one edge
added edge n1n2 from n1 to n2 in graph1

# Print the nodes contained in the graph and the
# edges outgoing from the parentNodes
graph1 contains: n1 n2
the children of n1 in graph1 are : n2(n1n2)